README.md

```diff
@@ -1,36 +1,48 @@
+
 # io-web-profile
 
-This is a [Next.js](https://nextjs.org/) project 
+`` 
+io-web-profile`` is the IO web platform that allows citizens to logout from the IO App session.
 
-## Getting Started
+This repository contains the code that composes the io-web-profile front-end.
 
-First, run the development server:
 
-```bash
-npm run dev
-# or
-yarn dev
-# or
-pnpm dev
-```
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+## Technologies
+[![My Skills](https://skillicons.dev/icons?i=ts,react,next)](https://skillicons.dev)
+
+This is Next.js 13 with app route project
+## Prerequisites
+In order to run the ``io-web-profile`` front-end locally you need the following tool installed on your machine.
+
+- ``Node.js 18 ``
```

Comment:
What you think about adding a .node-version file that can be used with nodenv?

Reply:
Created the file .node-version.